packages/wdio-utils/tests/utils.test.ts

```diff
@@ -191,9 +197,11 @@ describe('utils:isBase64', () => {
         expect(isBase64(inValidBase64)).toBe(false)
     })
     it('should throw if there no input to be checked', () => {
+        // @ts-ignore
```

Comment:
Can some of `@ts-ignore` in this files be removed?

Reply:
the types expect a string but we are testing here if the user passes in nothing how it will react so we have to ignore the types for these